backward.pc.in

```diff
@@ -0,0 +1,12 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=@CMAKE_INSTALL_PREFIX@
+libdir=${prefix}/lib
+includedir=${prefix}/include
+
+Name: @PROJECT_NAME@
+Description: Backward is a beautiful stack trace pretty printer for C++.
+Version: @PROJECT_VERSION@
+Requires: @PKGCONFIG_REQUIRES@
+Libs: -L${libdir} -l@TARGET_NAME@ @PKGCONFIG_LIBS@
+Cflags: -I${includedir} @PKGCONFIG_CFLAGS@
```

Comment:
I suspect adding @BACKWARD_DEFINITIONS@ to the cflags will add the proper defines instead of adding a `.hpp.in`. In both case this means the pkgfile is set to a specific list of libraries during its generation.